### ISSUE TITLE
EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/Instabug-iOS#171]
Hey there.. 
I received that in our crash reporter.

```
EXC_BAD_ACCESS Attempted to dereference garbage pointer 0x10. 
    Instabug/Helpers/IBGMethodSwizzler.m:212:9 -[IBGMethodSwizzler viewDidDisappear:]
    Frameworks/UIKit.framework/UIKit -[UINavigationController viewDidDisappear:]
    Frameworks/UIKit.framework/UIKit -[UIViewController _setViewAppearState:isAnimating:]
    Frameworks/UIKit.framework/UIKit -[UIViewController __viewDidDisappear:]
    Frameworks/UIKit.framework/UIKit ___64-[UIViewController viewDidMoveToWindow:shouldAppearOrDisappear:]_block_invoke.1271
    Frameworks/UIKit.framework/UIKit -[UIViewController _executeAfterAppearanceBlock]
    Frameworks/UIKit.framework/UIKit __runAfterCACommitDeferredBlocks
    Frameworks/UIKit.framework/UIKit __cleanUpAfterCAFlushAndRunDeferredBlocks
    Frameworks/UIKit.framework/UIKit __afterCACommitHandler
    Frameworks/CoreFoundation.framework/CoreFoundation ___CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
    Frameworks/CoreFoundation.framework/CoreFoundation ___CFRunLoopDoObservers
    Frameworks/CoreFoundation.framework/CoreFoundation ___CFRunLoopRun
    Frameworks/CoreFoundation.framework/CoreFoundation CFRunLoopRunSpecific
    PrivateFrameworks/GraphicsServices.framework/GraphicsServices GSEventRunModal
    Frameworks/UIKit.framework/UIKit UIApplicationMain
    AcuityApp mh_execute_header
    /usr/lib/system/libdyld.dylib start
```

```
mach
{
  "code": 16,
  "exception_name": "EXC_BAD_ACCESS",
  "subcode": 8,
  "exception": 1,
  "code_name": "KERN_INVALID_TASK"
}
signal
{
  "code": 0,
  "name": "SIGBUS",
  "signal": 10,
  "code_name": "BUS_NOOP"
}
type
mach
```

```
Thread 1·
/usr/lib/system/libsystem_kernel.dylib___semwait_signal	
/usr/lib/system/libsystem_c.dylibnanosleep	
Frameworks/Foundation.framework/Foundation+[NSThread sleepForTimeInterval:]	
Frameworks/Foundation.framework/Foundation-[__NSOperationInternal _start:]	
Frameworks/Foundation.framework/Foundation___NSOQSchedule_f	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 2·
/usr/lib/system/libsystem_kernel.dylib___psynch_rw_unlock	
/usr/lib/system/libsystem_pthread.dylibpthread_rwlock_unlock	
/usr/lib/libobjc.A.dylib__ZN9rwlock_ttILb0EE10unlockReadEv	
/usr/lib/libobjc.A.dyliblookUpImpOrForward	
/usr/lib/libobjc.A.dylib__objc_msgSend_uncached	
Frameworks/CoreFoundation.framework/CoreFoundation-[NSDictionary allValues]	
Frameworks/CoreData.framework/CoreData-[NSSQLEntity properties]	
Frameworks/CoreData.framework/CoreData-[NSSQLEntity manyToManyRelationships]	
Frameworks/CoreData.framework/CoreData-[NSSQLiteConnection createManyToManyTablesForEntity:]	
Frameworks/CoreData.framework/CoreData-[NSSQLiteConnection createTablesForEntities:]	
Frameworks/CoreData.framework/CoreData-[NSSQLiteConnection createSchema]	
Frameworks/CoreData.framework/CoreData-[NSSQLiteConnection connect]	
Frameworks/CoreData.framework/CoreData___32-[NSSQLCore _loadAndSetMetadata]_block_invoke	
Frameworks/CoreData.framework/CoreData___37-[NSSQLiteConnection performAndWait:]_block_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_invoke	
Frameworks/CoreData.framework/CoreData-[NSSQLiteConnection performAndWait:]	
Frameworks/CoreData.framework/CoreData-[NSSQLCore _loadAndSetMetadata]	
Frameworks/CoreData.framework/CoreData-[NSSQLCore loadMetadata:]	
Frameworks/CoreData.framework/CoreData___91-[NSPersistentStoreCoordinator addPersistentStoreWithType:configuration:URL:options:error:]_block_invoke	
Frameworks/CoreData.framework/CoreDatagutsOfBlockToNSPersistentStoreCoordinatorPerform	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_invoke	
Frameworks/CoreData.framework/CoreData__perform	
Frameworks/CoreData.framework/CoreData-[NSPersistentStoreCoordinator addPersistentStoreWithType:configuration:URL:options:error:]	
Instabug/Managers/IBGCoreDataPersistenceManager.m:159:20__51-[IBGCoreDataPersistenceManager initializeCoreData]_block_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_call_block_and_release	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 3·
/usr/lib/system/libsystem_kernel.dylib___psynch_rw_rdlock	
/usr/lib/system/libsystem_pthread.dylib__pthread_rwlock_lock	
/usr/lib/libobjc.A.dylib__ZN9rwlock_ttILb0EE4readEv	
/usr/lib/libobjc.A.dyliblookUpImpOrForward	
/usr/lib/libobjc.A.dylib__objc_msgSend_uncached	
Frameworks/CoreFoundation.framework/CoreFoundation-[NSDictionary countByEnumeratingWithState:objects:count:]	
AcuityAppmh_execute_header	
 AcuityAppmh_execute_header	
 Frameworks/Foundation.framework/Foundation___NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__	
Frameworks/Foundation.framework/Foundation-[NSBlockOperation main]	
Frameworks/Foundation.framework/Foundation-[__NSOperationInternal _start:]	
Frameworks/Foundation.framework/Foundation___NSOQSchedule_f	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 4·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopServiceMachPort	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopRun	
Frameworks/CoreFoundation.framework/CoreFoundationCFRunLoopRunSpecific	
Frameworks/Foundation.framework/Foundation-[NSRunLoop(NSRunLoop) runMode:beforeDate:]	
Frameworks/Foundation.framework/Foundation-[NSRunLoop(NSRunLoop) runUntilDate:]	
Frameworks/UIKit.framework/UIKit-[UIEventFetcher threadMain]	
Frameworks/Foundation.framework/Foundation___NSThread__start__	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 5·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
/usr/lib/system/libdispatch.dylib__dispatch_mach_send_and_wait_for_reply	
/usr/lib/system/libdispatch.dylibdispatch_mach_send_with_result_and_wait_for_reply	
/usr/lib/system/libxpc.dylibxpc_connection_send_message_with_reply_sync	
Frameworks/Security.framework/Securitysecurityd_message_with_reply_sync	
Frameworks/Security.framework/Securitysecurityd_send_sync_and_do	
Frameworks/Security.framework/Security___SecTrustEvaluateIfNecessary_block_invoke_3	
Frameworks/Security.framework/SecuritySecOSStatusWith	
Frameworks/Security.framework/Security___SecTrustEvaluateIfNecessary_block_invoke_2	
/usr/lib/system/libsystem_trace.dylib__os_activity_initiate_impl	
Frameworks/Security.framework/Security___SecTrustEvaluateIfNecessary_block_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_invoke	
Frameworks/Security.framework/SecuritySecTrustEvaluateIfNecessary	
Frameworks/Security.framework/SecuritySecTrustEvaluate	
Frameworks/CFNetwork.framework/CFNetwork__ZN14CFNetworkTrust14strictEvaluateEhN20StrictSecurityPolicy21SessionStrengthPolicyEm	
Frameworks/CFNetwork.framework/CFNetwork__ZN15TCPIOConnection29_tlsProcessTrust_PolicyStrictEv	
Frameworks/CFNetwork.framework/CFNetwork__ZN15TCPIOConnection28_tlsProcessTrustPolicyResultEP14nw_tls_context	
/usr/lib/system/libdispatch.dylib__dispatch_call_block_and_release	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 6·
/usr/lib/system/libsystem_kernel.dylib___workq_kernreturn	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 7·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopServiceMachPort	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopRun	
Frameworks/CoreFoundation.framework/CoreFoundationCFRunLoopRunSpecific	
Frameworks/CFNetwork.framework/CFNetwork+[NSURLConnection(Loader) _resourceLoadLoop:]	
Frameworks/Foundation.framework/Foundation___NSThread__start__	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 8·
/usr/lib/system/libsystem_kernel.dylib___bsdthread_ctl	
/usr/lib/system/libsystem_pthread.dylib__pthread_qos_override_end_direct	
/usr/lib/libobjc.A.dylib__ZN9rwlock_ttILb0EE10unlockReadEv	
/usr/lib/libobjc.A.dyliblookUpImpOrForward	
/usr/lib/libobjc.A.dylib__objc_msgSend_uncached	
AcuityAppmh_execute_header	
 AcuityAppmh_execute_header	
 AcuityAppmh_execute_header	
 /usr/lib/system/libdispatch.dylib__dispatch_call_block_and_release	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 9·
Thread 10·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopServiceMachPort	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopRun	
Frameworks/CoreFoundation.framework/CoreFoundationCFRunLoopRunSpecific	
AcuityApp__cxa_throw	
 Frameworks/Foundation.framework/Foundation___NSThread__start__	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 11·
/usr/lib/system/libsystem_kernel.dylib___semwait_signal	
/usr/lib/system/libsystem_c.dylibnanosleep	
/usr/lib/libc++.1.dylib__ZNSt3__111this_thread9sleep_forERKNS_6chrono8durationIxNS_5ratioILl1ELl1000000000EEEEE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZNSt3__111this_thread9sleep_forIxNS_5ratioILl1ELl1000EEEEEvRKNS_6chrono8durationIT_T0_EE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN7bmallocL14waitUntilFalseERNSt3__111unique_lockINS_11StaticMutexEEENS0_6chrono8durationIxNS0_5ratioILl1ELl1000EEEEERb	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN7bmalloc4Heap8scavengeERNSt3__111unique_lockINS_11StaticMutexEEENS1_6chrono8durationIxNS1_5ratioILl1ELl1000EEEEE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN7bmalloc4Heap18concurrentScavengeEv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN7bmalloc9AsyncTaskINS_4HeapEMS1_FvvEE13threadRunLoopEv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN7bmalloc9AsyncTaskINS_4HeapEMS1_FvvEE16threadEntryPointEPS4_	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZNSt3__114__thread_proxyINS_5tupleIJPFvPN7bmalloc9AsyncTaskINS2_4HeapEMS4_FvvEEEES8_EEEEEPvSC_	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 12·
/usr/lib/system/libsystem_kernel.dylib___psynch_rw_rdlock	
/usr/lib/system/libsystem_pthread.dylib__pthread_rwlock_lock	
/usr/lib/libobjc.A.dylib__ZN9rwlock_ttILb0EE4readEv	
/usr/lib/libobjc.A.dyliblookUpImpOrForward	
/usr/lib/libobjc.A.dylib__objc_msgSend_uncached	
Frameworks/CoreFoundation.framework/CoreFoundation-[CFPrefsPlistSource alreadylocked_setValues:forKeys:count:]	
Frameworks/CoreFoundation.framework/CoreFoundation-[CFPrefsSource setValues:forKeys:count:removeValuesForKeys:count:]	
Frameworks/CoreFoundation.framework/CoreFoundation-[CFPrefsSearchListSource alreadylocked_setValues:forKeys:count:]	
Frameworks/CoreFoundation.framework/CoreFoundation-[CFPrefsSource setValues:forKeys:count:removeValuesForKeys:count:]	
Frameworks/CoreFoundation.framework/CoreFoundation-[CFPrefsSource setValue:forKey:]	
Frameworks/CoreFoundation.framework/CoreFoundation___108-[_CFXPreferences(SearchListAdditions) withSearchListForIdentifier:container:cloudConfigurationURL:perform:]_block_invoke	
Frameworks/CoreFoundation.framework/CoreFoundationnormalizeQuintuplet	
Frameworks/CoreFoundation.framework/CoreFoundation-[_CFXPreferences(SearchListAdditions) withSearchListForIdentifier:container:cloudConfigurationURL:perform:]	
Frameworks/CoreFoundation.framework/CoreFoundation-[_CFXPreferences setValue:forKey:appIdentifier:container:configurationURL:]	
Frameworks/Foundation.framework/Foundation-[NSUserDefaults(NSUserDefaults) setObject:forKey:]	
Instabug/Defaults/IBGFeatures.m:231:9-[IBGFeatures setFeatureOfName:toValue:isUserSet:]	
Instabug/Defaults/IBGFeatures.m:149:9-[IBGFeatures JSONSetFeature:toValue:]	
Instabug/Defaults/IBGFeatures.m:181:9-[IBGFeatures featuresRequestDone:isSuccess:response:error:]	
Instabug/Server and Requests/Specific Requests/IBGFeaturesRequest.m:74:9-[IBGFeaturesRequest dataServerExecutionDone:statusCode:error:response:]	
Instabug/Server and Requests/Generic Requests/IBGServerDataRequest.m:153:47__75+[IBGServerDataRequest executeWithURL:HTTPMethod:payload:headers:delegate:]_block_invoke_2	
Instabug/Server and Requests/Generic Requests/IBGServerDataRequest.m:121:5-[IBGServerDataRequest prepareResponseWithData:error:rawResponse:completionHandler:]	
Instabug/Server and Requests/Generic Requests/IBGServerDataRequest.m:149:30__75+[IBGServerDataRequest executeWithURL:HTTPMethod:payload:headers:delegate:]_block_invoke	
Frameworks/CFNetwork.framework/CFNetwork___67+[NSURLConnection sendAsynchronousRequest:queue:completionHandler:]_block_invoke_2	
Frameworks/Foundation.framework/Foundation___NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__	
Frameworks/Foundation.framework/Foundation-[NSBlockOperation main]	
Frameworks/Foundation.framework/Foundation-[__NSOperationInternal _start:]	
Frameworks/Foundation.framework/Foundation___NSOQSchedule_f	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 13·
/usr/lib/system/libsystem_kernel.dylib___psynch_cvwait	
/usr/lib/system/libsystem_pthread.dylib__pthread_cond_wait	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF15ThreadCondition9timedWaitERNS_5MutexEd	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF10ParkingLot21parkConditionallyImplEPKvRKNS_12ScopedLambdaIFbvEEERKNS3_IFvvEEERKNS_24TimeWithDynamicClockTypeE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF13ConditionBase9waitUntilINS_4LockEEEbRT_RKNS_24TimeWithDynamicClockTypeE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZNSt3__110__function6__funcIZN3WTF15AutomaticThread5startERKNS2_6LockerINS2_8LockBaseEEEE3$_0NS_9allocatorIS9_EEFvvEEclEv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTFL16threadEntryPointEPv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTFL19wtfThreadEntryPointEPv	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 14·
/usr/lib/system/libsystem_kernel.dylib___psynch_cvwait	
/usr/lib/system/libsystem_pthread.dylib__pthread_cond_wait	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF15ThreadCondition9timedWaitERNS_5MutexEd	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF10ParkingLot21parkConditionallyImplEPKvRKNS_12ScopedLambdaIFbvEEERKNS3_IFvvEEERKNS_24TimeWithDynamicClockTypeE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTF13ConditionBase9waitUntilINS_4LockEEEbRT_RKNS_24TimeWithDynamicClockTypeE	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZNSt3__110__function6__funcIZN3WTF15AutomaticThread5startERKNS2_6LockerINS2_8LockBaseEEEE3$_0NS_9allocatorIS9_EEFvvEEclEv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTFL16threadEntryPointEPv	
Frameworks/JavaScriptCore.framework/JavaScriptCore__ZN3WTFL19wtfThreadEntryPointEPv	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 15·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopServiceMachPort	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopRun	
Frameworks/CoreFoundation.framework/CoreFoundationCFRunLoopRunSpecific	
Frameworks/AVFoundation.framework/Frameworks/AVFAudio.framework/AVFAudio__ZN20GenericRunLoopThread5EntryEPv	
Frameworks/AVFoundation.framework/Frameworks/AVFAudio.framework/AVFAudio__ZN9CAPThread5EntryEPS_	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 16·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
/usr/lib/system/libsystem_kernel.dylibthread_suspend	
AcuityApp__cxa_throw	
 /usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 17·
Thread 18·
/usr/lib/system/libsystem_kernel.dylibmach_msg_trap	
/usr/lib/system/libsystem_kernel.dylibmach_msg	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopServiceMachPort	
Frameworks/CoreFoundation.framework/CoreFoundation___CFRunLoopRun	
Frameworks/CoreFoundation.framework/CoreFoundationCFRunLoopRunSpecific	
Frameworks/CFNetwork.framework/CFNetwork__ZL27_privateRunloopEmulationSetPv	
/usr/lib/system/libsystem_pthread.dylib__pthread_body	
/usr/lib/system/libsystem_pthread.dylib__pthread_start	
Thread 19·
/usr/lib/system/libsystem_kernel.dylib___ulock_wait	
/usr/lib/system/libdispatch.dylib__dispatch_ulock_wait	
/usr/lib/system/libdispatch.dylib__dispatch_thread_event_wait_slow	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_slow	
Frameworks/CoreData.framework/CoreData__perform	
Frameworks/CoreData.framework/CoreData-[NSManagedObjectContext(_NestedContextSupport) executeRequest:withContext:error:]	
Frameworks/CoreData.framework/CoreData-[NSManagedObjectContext save:]	
Instabug/Logging/IBGEventLogger.m:50:9__45-[IBGEventLogger addEventLogWithName:params:]_block_invoke	
Instabug/IBGDALService.m:51:17__44-[IBGDALService writeASync:completionBlock:]_block_invoke_3	
Frameworks/CoreData.framework/CoreDatadeveloperSubmittedBlockToNSManagedObjectContextPerform	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_invoke	
Frameworks/CoreData.framework/CoreData-[NSManagedObjectContext performBlockAndWait:]	
Instabug/IBGDALService.m:50:13__44-[IBGDALService writeASync:completionBlock:]_block_invoke_2	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_barrier_sync_f_invoke	
Instabug/IBGDALService.m:68:13__44-[IBGDALService writeASync:completionBlock:]_block_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_call_block_and_release	
/usr/lib/system/libdispatch.dylib__dispatch_client_callout	
/usr/lib/system/libdispatch.dylib__dispatch_queue_serial_drain	
/usr/lib/system/libdispatch.dylib__dispatch_queue_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_queue_override_invoke	
/usr/lib/system/libdispatch.dylib__dispatch_root_queue_drain	
/usr/lib/system/libdispatch.dylib__dispatch_worker_thread3	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 20·
Thread 21·
/usr/lib/system/libsystem_kernel.dylib___workq_kernreturn	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 22·
/usr/lib/system/libsystem_kernel.dylib___workq_kernreturn	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread	
Thread 23·
/usr/lib/system/libsystem_kernel.dylib___workq_kernreturn	
/usr/lib/system/libsystem_pthread.dylib__pthread_wqthread
```

I'm using react-native-instabug and I'm installing Instabug via cocoapods like so:

pod 'Instabug', '~> 7.3.2'

**Comment by Kmohamed** [comment]
@Sudoplz could you please make a issue on support and I will follow up with you.

**Comment by Kmohamed** [comment]
This issue is Fixed on 7.3.5 also 🙈
